integration/spark/src/main/scala/org/apache/carbondata/spark/rdd/CarbonMergerRDD.scala

```diff
@@ -102,6 +102,11 @@ class CarbonMergerRDD[K, V](
         var dataloadStatus = CarbonCommonConstants.STORE_LOADSTATUS_FAILURE
         val carbonSparkPartition = theSplit.asInstanceOf[CarbonSparkPartition]
 
+        // get destination segment properties as sent from driver which is of last segment.
+
+        val segmentProperties = new SegmentProperties(carbonMergerMapping.columnSchemaList.asJava,
```

Comment:
rename it to maxSegmentcolumnSchemaList and maxSegmentColCardinality.
Write comment where it is declared